client-js/main/client/abstract-client.js

```diff
@@ -120,47 +98,34 @@ export class AbstractClient extends Client {
   /**
    * @inheritDoc
    */
-  subscribeToEntities({ofType: type, byIds: ids, byId: id}) {
-    if (typeof ids !== 'undefined' && typeof id !== 'undefined') {
-      throw new Error('No entity IDs set. Specify either a single entity ID or' +
-          ' multiple entity IDs to subscribe to the entity state updates.');
+  fetch({entity: cls, byIds: ids, byId: id}) {
```

Comment:
Having both `byId` and `byIds` looks confusing to me. Can we leave just a `byIds` clause and detect whether an array or a single value is passed?